actions/actions.py

```diff
@@ -484,8 +484,11 @@ def run(
                 intent_ranking = intent_ranking[:1]
         first_intent_names = [
             intent.get("name", "")
+            if intent.get("name", "") not in ["out_of_scope", "faq", "chitchat"]
+            else tracker.latest_message.get("response_selector")
+            .get(intent.get("name", ""))
+            .get("full_retrieval_intent")
```

Comment:
maybe it makes sense to make a little helper method for this that's more intuitive with a proper if/else so that the next person trying to fix this isn't gonna want to grumpily @ you 

Also is there an easy way for us to automate getting a list of response selector intents so we don't have to update that? maybe we can reuse some rasa code?

Reply:
hm but then again we don't want to have to import the huge rasa package to do so

Reply:
yeah let's not :D not sure it's worth making a helper method, but I can leave a comment on what is happening here

Reply:
I think a helper method would be better as it wouldn't necessitate a 4 line comment, but whatever 😄i might end up changing it next time i edit the actions though 😉

Reply:
then the helper method would have a 4 line comment 😂